Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1873,6 +1873,7 @@
         SYS_getrlimit
         SYS_kdebug_trace
         SYS_pathconf
+        SYS_quotactl
```

Comment:
Is this only needed for WebConent, and not any other services?

Reply:
> Is this only needed for WebConent, and not any other services?

That is a good point! I will look into that. I have also updated the PR, since this sys call is only needed on Intel.

Thanks for reviewing!